app/helpers/is-access-level-in-effect.js

```diff
@@ -0,0 +1,82 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+import { isPresent } from '@ember/utils';
+import CONSTANTS from 'frontend-kaleidos/config/constants';
+
+export default class IsAccessLevelInEffect extends Helper {
+  @service agendaService;
+
+  async compute([accessLevel, piece]) {
+    if (accessLevel.uri === CONSTANTS.ACCESS_LEVELS.INTERN_SECRETARIE) {
+      // Always in effect
+      return true;
+    }
+
+    let inEffect = false;
+    if (piece) {
+      // Always assume that we're dealing with the latest agenda
+      const agendaitems = (await piece.agendaitems).toArray();
```

Comment:
Not a big fan of using the inverse relation between models and pieces in general with a many to many.
this inverse is 'read-only' (not serialized) and the `inverse: null` prevents ember from updating agendaitems.pieces in the store. So technically this should be fine, but it's possible some concurrency issues could ensue.